core/templates/dev/head/components/collection_summary_tile_directive.html

```diff
@@ -0,0 +1,52 @@
+<script type="text/ng-template" id="summaryTile/collection">
+  <md-card class="oppia-gallery-summary-tile bottom-card">
+    <div class="top-section" style="background-color: <[getThumbnailBgColor()]>;">
+    </div>
+    <md-card class="oppia-gallery-summary-tile middle-card">
+      <div class="top-section" style="background-color: <[getThumbnailBgColor()]>;">
+      </div>
+      <md-card class="oppia-gallery-summary-tile top-card">
+        <a ng-href="<[getCollectionLink()]>">
+          <div class="top-section" style="background-color: <[getThumbnailBgColor()]>;">
+            <img class="thumbnail-image" ng-src="<[getThumbnailIconUrl()]>">
+            <h2 class="title"><[getCollectionTitle()]></h2>
+          </div>
+
+          <div>
+            <div class="objective">
+              <[getObjective() | truncateAndCapitalize: 45]>
+              <span ng-if="!getObjective()">No objective specified.</span>
+            </div>
+
+            <ul layout="row" class="metrics" layout-align="space-between center">
+              <li>
+                <span>
+                  COLLECTIONS(<[getNumExplorations()]>)
```

Comment:
I think to match the mock, this should be ``COLLECTION (<[getNumExplorations()]>)``